{"name": "测试用例描述：错误的密码登录失败", "status": "passed", "description": "测试描述：错误的密码登录失败", "steps": [{"name": "输入错误的密码", "status": "passed", "attachments": [{"name": "错误的密码", "source": "0827abb2-c5d8-4998-be3f-9ec1ae94b713-attachment.attach"}], "start": 1651989191387, "stop": 1651989191387}], "attachments": [{"name": "截图", "source": "e8779c60-5e4d-4f43-9e45-95f8f29f1582-attachment.png", "type": "image/png"}, {"name": "截图", "source": "c22f2dfb-c01c-4650-81c6-43c8eed5cce4-attachment.png", "type": "image/png"}], "start": 1651989191387, "stop": 1651989193555, "uuid": "fddaf9a2-e68e-4730-977a-71ad3a7a7aac", "historyId": "e042998cd34bf7d92d802f115102d44e", "testCaseId": "ce17e49339328b1be4c6805d1bcb9a3f", "fullName": "test_two.Test#test_22", "labels": [{"name": "story", "value": "登录"}, {"name": "epic", "value": "xxx测试"}, {"name": "severity", "value": "critical"}, {"name": "feature", "value": "登录模块"}, {"name": "suite", "value": "test_two"}, {"name": "subSuite", "value": "Test"}, {"name": "host", "value": "MS-SLSXLICKSUSF"}, {"name": "thread", "value": "14312-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_two"}]}